recipes/sole/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class SoleConan(ConanFile):
+    name = "sole"
+    homepage = "https://github.com/r-lyeh-archived/sole"
+    description = "Sole is a lightweight C++11 library to generate universally unique identificators (UUID), both v1 and v4."
+    topics = ("conan", "uuid", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler"
+    no_copy_source = True
+    license = "zlib"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_name = "sole-" + self.version
+        os.rename(extracted_name, self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses",
+                  src=self._source_subfolder)
+        self.copy(pattern="sole.hpp", dst="include",
+                  src=self._source_subfolder)
+
+    def package_info(self):
+        if str(self.settings.os) in ["Linux"]:
```

Comment:
it's enough to have:

```suggestion
        if self.settings.os == "Linux":
```